samples/property/provider/src/main.rs

```diff
@@ -60,31 +66,41 @@ async fn register_ambient_air_temperature(
     Ok(())
 }
 
+fn create_property(ambient_air_temperature: i32) -> String {
+    let metadata = Metadata { model: sdv::hvac::ambient_air_temperature::ID.to_string() };
+
+    let property: Property = Property { ambient_air_temperature, metadata };
```

Comment:
the type annotation here is not necessary

Reply:
I prefer to make it explicit. Leaving it as is.